bin/install_windows_10_sdk.ps1

```diff
@@ -25,14 +25,7 @@ Write-Output "--- :windows: Installing Windows 10 SDK and Visual Studio Build To
 
 # See list at https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022
 $allowedVersions = @(
-  "20348",
-  "19041",
-  "18362",
-  "17763",
-  "17134",
-  "16299",
-  "15063",
-  "14393"
+  "19041"
```

Comment:
❓  Are we sure all those other versions should be removed from the allowlist, not just 20348?

In other words, have you tested each single one of those and removed them all because you validated that `19041` was indeed the only one remaining valid? Or did you remove them all just because you considered it was not worth the effort for us to allow/support those older versions and didn't want to bother?

(I'm fine with either, just curious)